xml/System.Diagnostics/Activity.xml

```diff
@@ -224,8 +224,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>This is an ID that is specific to a particular request. Filtering to a particular ID insures that you get only one request that matches. Id has a hierarchical structure: '|root-id.id1_id2.id3_' Id is generated when <see cref="M:System.Diagnostics.Activity.Start" /> is called by appending suffix to <see cref="P:System.Diagnostics.Activity.Parent" />.Id or <see cref="P:System.Diagnostics.Activity.ParentId" />; <see cref="T:System.Diagnostics.Activity" /> has no Id until it starts. See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format" /> for more details.</summary>
```

Comment:
All instances of `ID` should be capitalized

Reply:
You should replace `insure` with `ensure`

Reply:
```suggestion
        <summary>This is an ID that is specific to a particular request.
Filtering to a particular ID ensures that you get only one request that matches.
An ID has a hierarchical structure: '|root-id.id1_id2.id3_'.
The ID is generated when <see cref="M:System.Diagnostics.Activity.Start" /> is called by appending suffix to Parent.Id or <see cref="P:System.Diagnostics.Activity.ParentId" />.
An <see cref="T:System.Diagnostics.Activity" /> has no ID until it starts.
See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format" /> for more details.</summary>
```